packages/functional-tests/tests/react-conversion/oauthSignup.spec.ts

```diff
@@ -97,49 +100,23 @@ test.describe('severity-1 #smoke', () => {
       await relier.signOut();
     });
 
-    test('signup oauth webchannel - sync mobile or FF desktop 123+', async ({
+    test('signup oauth webchannel with Sync - FF desktop 123+', async ({
```

Comment:
nit: we should remove `FF desktop 123+`

Also - should we have a separate test for oauth sync mobile and oauth sync desktop? I don't have a strong preference since they go through the same flow, but we import both sets of query params and I think the test name should reflect whatever query params are used.

Reply:
I don't believe we need too? With the nightly and oauth flow enabled I couldn't get the webchannel events to be mocked so it wouldn't really be able to test the mobile flow either.

Reply:
That seems fine then, but we've got unused imports `syncMobileOAuthQueryParams` / `FF_OAUTH_CLIENT_ID` btw.